src/app/common/ResourceCodeBlock.vue

```diff
@@ -76,3 +93,11 @@ function toYamlRepresentation(resource: Entity): string {
   return toYaml(resourceWithoutTimes)
 }
 </script>
+
+<style lang="scss">
+.kubernetes-copy-button:not(.increase-specificity.increase-specificity) {
```

Comment:
😬 , what's the background here? I assume something else is already overwriting with `.increase-specificity` somehow?